test/sailci.test.js

```diff
@@ -0,0 +1,23 @@
+import test from 'ava';
```

Comment:
Similarly the file should be named `test/sail.test.js` rather than `test/sailci.test.js`.